resources/warnings.json

```
@@ -19106,7 +19106,7 @@
     "versions": [
       {
         "lastVersion": "239.v5f5c278708cf",
-        "pattern": ".*"
+        "pattern": "(1|237|239[.]v5f5c278708cf)(|[.-].+)"
```

```suggestion
        "pattern": "(1|237|239)(|[.-].+)"
```
Hash suffix is only necessary if there's a JEP-229 backport (like `239.241.vdeadbeef`).